samples/excel/01-basics/api-call-with-notification-toast.yaml

```diff
@@ -7,9 +7,7 @@ host: EXCEL
 api_set:
     ExcelApi: 1.1
 script:
-    content: |
-        declare var fabric: any;
-
+    content: |-
```

Comment:
when you start customizing it will be a pain.. and folks will struggle. so i'd vote for leaving it in.